[2023-08-30T00:00:01.724+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T00:00:01.731+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T00:00:01.737+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T00:00:01.738+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T00:00:01.746+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T00:00:01.751+0000] {standard_task_runner.py:57} INFO - Started process 8276 to run task
[2023-08-30T00:00:01.754+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp4tr8wma9']
[2023-08-30T00:00:01.758+0000] {standard_task_runner.py:85} INFO - Job 40: Subtask import_data_to_csv
[2023-08-30T00:00:01.773+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T00:00:01.809+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host e69093e17de8
[2023-08-30T00:00:01.821+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T00:00:01.869+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T00:00:01.871+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T00:00:01.873+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T00:00:01.881+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T00:00:01.947+0000] {subprocess.py:93} INFO - python: can't open file '/Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-30T00:00:01.951+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-30T00:00:01.962+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-30T00:00:01.966+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T000001, end_date=20230830T000001
[2023-08-30T00:00:01.978+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 40 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 8276)
[2023-08-30T00:00:02.009+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-30T00:00:02.029+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T18:56:46.306+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.317+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T18:56:46.328+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T18:56:46.331+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T18:56:46.354+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T18:56:46.358+0000] {standard_task_runner.py:57} INFO - Started process 550 to run task
[2023-08-30T18:56:46.366+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpezq0npa2']
[2023-08-30T18:56:46.370+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask import_data_to_csv
[2023-08-30T18:56:46.401+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.469+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host 5e7fbf8b7901
[2023-08-30T18:56:46.489+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T18:56:46.589+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T18:56:46.591+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T18:56:46.593+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T18:56:46.613+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T18:56:46.704+0000] {subprocess.py:93} INFO - python: can't open file '/Users/jinee.hira/Dropbox/virtual_envs/austin-flow/***/dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-30T18:56:46.715+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-30T18:56:46.730+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-30T18:56:46.736+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T185646, end_date=20230830T185646
[2023-08-30T18:56:46.760+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 14 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 550)
[2023-08-30T18:56:46.783+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-30T18:56:46.812+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:20:00.552+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.566+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:20:00.575+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:20:00.577+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:20:00.594+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:20:00.599+0000] {standard_task_runner.py:57} INFO - Started process 270 to run task
[2023-08-30T19:20:00.610+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpdyydpvgf']
[2023-08-30T19:20:00.615+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask import_data_to_csv
[2023-08-30T19:20:00.641+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.691+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host 884d86529f65
[2023-08-30T19:20:00.722+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:20:00.798+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:20:00.800+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:20:00.802+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python ./dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T19:20:00.829+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:20:00.945+0000] {subprocess.py:93} INFO - python: can't open file './dags/data_ingestion.py': [Errno 2] No such file or directory
[2023-08-30T19:20:00.953+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-08-30T19:20:00.978+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-08-30T19:20:00.985+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T192000, end_date=20230830T192000
[2023-08-30T19:20:01.010+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 13 for task import_data_to_csv (Bash command failed. The command returned a non-zero exit code 2.; 270)
[2023-08-30T19:20:01.060+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-08-30T19:20:01.093+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:31:37.210+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:37.219+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:31:37.232+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:31:37.233+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:31:37.249+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:31:37.255+0000] {standard_task_runner.py:57} INFO - Started process 267 to run task
[2023-08-30T19:31:37.259+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpyezwwk6t']
[2023-08-30T19:31:37.268+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask import_data_to_csv
[2023-08-30T19:31:37.288+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:37.352+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host f6bcf103ff79
[2023-08-30T19:31:37.378+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:31:37.460+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:31:37.462+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:31:37.464+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T19:31:37.486+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:31:42.084+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:31:42.125+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T193137, end_date=20230830T193142
[2023-08-30T19:31:42.172+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:31:42.192+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:41:53.574+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.588+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:41:53.602+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:41:53.603+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:41:53.626+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:41:53.632+0000] {standard_task_runner.py:57} INFO - Started process 331 to run task
[2023-08-30T19:41:53.643+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpvj3atwkr']
[2023-08-30T19:41:53.650+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask import_data_to_csv
[2023-08-30T19:41:53.674+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.723+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host 81eeb4b7c8a9
[2023-08-30T19:41:53.741+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:41:53.833+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:41:53.836+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:41:53.838+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T19:41:53.860+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:41:59.748+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:41:59.784+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T194153, end_date=20230830T194159
[2023-08-30T19:41:59.834+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:41:59.854+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:50:53.721+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.738+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:50:53.763+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:50:53.767+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:50:53.808+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:50:53.814+0000] {standard_task_runner.py:57} INFO - Started process 308 to run task
[2023-08-30T19:50:53.823+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpyytptiqv']
[2023-08-30T19:50:53.830+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask import_data_to_csv
[2023-08-30T19:50:53.857+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:53.908+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host 206ecb1dc8df
[2023-08-30T19:50:53.932+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:50:54.017+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:50:54.020+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:50:54.021+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T19:50:54.032+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:51:01.814+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:51:01.889+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T195053, end_date=20230830T195101
[2023-08-30T19:51:01.967+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:51:02.024+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T19:54:26.262+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.274+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:54:26.282+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T19:54:26.284+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T19:54:26.305+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T19:54:26.311+0000] {standard_task_runner.py:57} INFO - Started process 262 to run task
[2023-08-30T19:54:26.315+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpd3dlttdf']
[2023-08-30T19:54:26.319+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask import_data_to_csv
[2023-08-30T19:54:26.337+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.390+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host 86f9e97d6c70
[2023-08-30T19:54:26.411+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T19:54:26.480+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T19:54:26.484+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T19:54:26.486+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T19:54:26.500+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T19:54:30.533+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T19:54:30.565+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T195426, end_date=20230830T195430
[2023-08-30T19:54:30.617+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T19:54:30.661+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:05:49.241+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.251+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:05:49.264+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:05:49.266+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:05:49.281+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T20:05:49.291+0000] {standard_task_runner.py:57} INFO - Started process 467 to run task
[2023-08-30T20:05:49.297+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp45849m2f']
[2023-08-30T20:05:49.300+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask import_data_to_csv
[2023-08-30T20:05:49.321+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.365+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host 3938d7abd9ff
[2023-08-30T20:05:49.381+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:05:49.463+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T20:05:49.469+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:05:49.470+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T20:05:49.483+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:05:56.246+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:05:56.315+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T200549, end_date=20230830T200556
[2023-08-30T20:05:56.381+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:05:56.466+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:33:50.109+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.121+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:33:50.136+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:33:50.137+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:33:50.160+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T20:33:50.180+0000] {standard_task_runner.py:57} INFO - Started process 284 to run task
[2023-08-30T20:33:50.185+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp1mcxl6dj']
[2023-08-30T20:33:50.191+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask import_data_to_csv
[2023-08-30T20:33:50.219+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.283+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host 961f38e8363e
[2023-08-30T20:33:50.307+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:33:50.390+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T20:33:50.393+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:33:50.394+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T20:33:50.406+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:33:56.197+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:33:56.266+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T203350, end_date=20230830T203356
[2023-08-30T20:33:56.325+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:33:56.355+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T20:48:23.529+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.543+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:48:23.554+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T20:48:23.555+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T20:48:23.569+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T20:48:23.580+0000] {standard_task_runner.py:57} INFO - Started process 340 to run task
[2023-08-30T20:48:23.593+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmpmpnbq229']
[2023-08-30T20:48:23.597+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask import_data_to_csv
[2023-08-30T20:48:23.620+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.672+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host 8f66711a2898
[2023-08-30T20:48:23.689+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T20:48:23.771+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T20:48:23.774+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T20:48:23.775+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T20:48:23.793+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T20:48:30.694+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T20:48:30.751+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T204823, end_date=20230830T204830
[2023-08-30T20:48:30.788+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T20:48:30.815+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:07:36.647+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:36.666+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T21:07:36.680+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T21:07:36.682+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:07:36.704+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T21:07:36.713+0000] {standard_task_runner.py:57} INFO - Started process 297 to run task
[2023-08-30T21:07:36.720+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp7n3ny5_4']
[2023-08-30T21:07:36.725+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask import_data_to_csv
[2023-08-30T21:07:36.764+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:36.838+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host cbc745b7550b
[2023-08-30T21:07:36.861+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:07:37.059+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T21:07:37.076+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:07:37.085+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T21:07:37.116+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:07:52.887+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:07:52.928+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T210736, end_date=20230830T210752
[2023-08-30T21:07:52.985+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:07:53.037+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-08-30T21:17:04.204+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.226+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T21:17:04.241+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T21:17:04.244+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-30T21:17:04.268+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): import_data_to_csv> on 2023-08-29 00:00:00+00:00
[2023-08-30T21:17:04.274+0000] {standard_task_runner.py:57} INFO - Started process 352 to run task
[2023-08-30T21:17:04.285+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'traffic_report', 'import_data_to_csv', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/traffic_dag.py', '--cfg-path', '/tmp/tmp099zhg15']
[2023-08-30T21:17:04.293+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask import_data_to_csv
[2023-08-30T21:17:04.324+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.385+0000] {task_command.py:415} INFO - Running <TaskInstance: traffic_report.import_data_to_csv scheduled__2023-08-29T00:00:00+00:00 [running]> on host 5b9f14bf20e0
[2023-08-30T21:17:04.410+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-08-30T21:17:04.507+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='traffic_report' AIRFLOW_CTX_TASK_ID='import_data_to_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T00:00:00+00:00'
[2023-08-30T21:17:04.510+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-30T21:17:04.511+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/data_ingestion.py --date 2023-08-29']
[2023-08-30T21:17:04.522+0000] {subprocess.py:86} INFO - Output:
[2023-08-30T21:17:13.025+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-30T21:17:13.062+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=traffic_report, task_id=import_data_to_csv, execution_date=20230829T000000, start_date=20230830T211704, end_date=20230830T211713
[2023-08-30T21:17:13.104+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T21:17:13.127+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
